app/provider/Registry/ServiceProvider.php

```diff
@@ -41,6 +41,16 @@ class ServiceProvider extends AbstractServiceProvider
         'public_path' => BASE_PATH . '/public/',
         'tests_data' => BASE_PATH . '/tests/_data/',
         'tests_fixtures' => BASE_PATH . '/tests/_fixtures/',
+        'assets_path' => BASE_PATH . '/public/assets/',
+        'storage_path' => BASE_PATH . '/storage/',
+        'log_path' => BASE_PATH . '/storage/logs/',
+        'pids_path' => BASE_PATH . '/storage/pids/',
+        'annotations_cache' => BASE_PATH . '/storage/cache/annotations/',
+        'config_cache' => BASE_PATH . '/storage/cache/config/',
+        'data_cache' => BASE_PATH . '/storage/cache/data/',
+        'metaData_cache' => BASE_PATH . '/storage/cache/metaData/',
+        'views_cache' => BASE_PATH . '/storage/cache/views/',
+        'volt_cache' => BASE_PATH . '/storage/cache/volt/',
```

Comment:
could you create a `paths` key with `(object) ['views' => '....', 'public' => '...', '...' => '']`